dockstore-webservice/src/main/java/io/dockstore/webservice/resources/HostedWorkflowResource.java

```diff
@@ -177,10 +183,15 @@ public Workflow editHosted(User user, Long entryId, Set<SourceFile> sourceFiles)
     @Path("/hostedEntry/{entryId}")
     @Timed
     @UnitOfWork
-    @ApiOperation(nickname = "addZip", value = "Creates a new revision of a hosted workflow from a zip",
-            authorizations = {@Authorization(value = JWT_SECURITY_DEFINITION_NAME)}, response = Workflow.class)
-    public Workflow addZip(@ApiParam(hidden = true) @Auth User user, @ApiParam(value = "hosted entry ID")
-        @PathParam("entryId") Long entryId,  @FormDataParam("file") InputStream payload) {
+    @ApiOperation(nickname = "addZip", value = "Creates a new revision of a hosted workflow from a zip", authorizations = {
+        @Authorization(value = JWT_SECURITY_DEFINITION_NAME)}, response = Workflow.class)
+    @Deprecated(since = "1.9.0")
+    @Operation(operationId = "addZip", summary = "Creates a new revision of a hosted workflow from a zip", security = @SecurityRequirement(name = "bearer"), deprecated = true)
```

Comment:
`name = "bearer"` can probably be JWT_SECURITY_DEFINITION_NAME

Reply:
I tried `name = JWT_SECURITY_DEFINITION_NAME`, but openapi.yaml doesn't seem to like that because the authorization pop up looks like this:
![Screenshot from 2020-02-14 09-25-44](https://user-images.githubusercontent.com/25287123/74539429-02005700-4f0c-11ea-8aad-bc49e9ac2628.png).

`name = "bearer"` looks like this:
![Screenshot from 2020-02-14 09-27-21](https://user-images.githubusercontent.com/25287123/74539553-396f0380-4f0c-11ea-9ce8-c703e2244696.png)

This is probably because openapi.yaml is case-sensitive.

Reply:
What's the openapi.yaml diff?

Reply:
the only difference is that for `name = "bearer"`, openapi.yaml looks like:
security:
      - bearer: []

and for `name = JWT_SECURITY_DEFINITION_NAME`, openapi.yaml looks like:
security:
      - BEARER: []

Reply:
Ahh, I understand the issue now. Though this is out of the scope for the PR.

1. The security scheme is defined multiple times (redundantly)
2. The name is different from Swagger (slightly annoying)
3. The name is not a constant somewhere (so easy to get the case wrong).

I'll make a SEAB issue later.